src/libraries/System.Private.CoreLib/src/System/Text/StringBuilder.cs

```diff
@@ -1841,6 +1841,17 @@ public StringBuilder AppendFormat(IFormatProvider? provider, CompositeFormat for
         /// </remarks>
         public StringBuilder Replace(string oldValue, string? newValue) => Replace(oldValue, newValue, 0, Length);
 
+        /// <summary>
+        /// Replaces all instances of one read-only character span with another in this builder.
```

Comment:
I am not very big fan of this wording, but since it's consistent with other `Replace` overloads I am fine with that.

Reply:
I've attempted to keep it similar to the existing code/comments. It could probably be better, but went for consistency instead.